{"name": "test_one[nag]", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntest_calc.py:21: AssumptionFailure\n>>\tpytest.assume(result == get_add_dates[2])\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               test_calc.py:21: AssumptionFailure\nE               >>\tpytest.assume(result == get_add_dates[2])\nE               AssertionError: assert False\n\n..\\..\\..\\AppData\\Roaming\\Python\\Python38\\site-packages\\six.py:702: FailedAssumption"}, "steps": [{"name": "此处为第二个执行", "status": "passed", "start": 1607844300472, "stop": 1607844300472}], "attachments": [{"name": "stdout", "source": "617bc944-7d3d-4d50-a74a-7a2cd561d43b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_add_dates", "value": "[0.1, 0.2, 0.3]"}], "start": 1607844300471, "stop": 1607844300470, "uuid": "5f006ca4-16d6-4f20-a003-c287d9aff68c", "historyId": "0c951cdb497480a7d7b96944793f09ab", "testCaseId": "5b53a6327a2dd636b4220af42ad90cc9", "fullName": "test_calc.TestCalc#test_one", "labels": [{"name": "feature", "value": "类的allure"}, {"name": "tag", "value": "flaky(reruns=1, reruns_delay=1)"}, {"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "add"}, {"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "LAPTOP-2NSHDHR9"}, {"name": "thread", "value": "4024-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}